tests/src/test/scala/com/nvidia/spark/rapids/HashAggregatesSuite.scala

```diff
@@ -34,11 +35,15 @@ class HashAggregatesSuite extends SparkQueryCompareTestSuite {
     conf.set("spark.rapids.sql.hashAgg.replaceMode", mode)
   }
 
-  private def checkExecNode(result: DataFrame): Unit = {
-    if (result.queryExecution.executedPlan.conf.getAllConfs(RapidsConf.SQL_ENABLED.key).toBoolean) {
-      assert(result.queryExecution.executedPlan.find {
-        _.isInstanceOf[GpuHashAggregateExec]
-      }.isDefined, "as the GPU plan expected a GPU aggregate but did not find any!")
+  private def checkExecPlan(plan: SparkPlan): Unit = {
```

Comment:
Ok, this makes sense. I do wonder what happens if AQE runs multiple times, but that's a general AQE question.